docs/manual/guides/mumble.de.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Mumble Installation"
+description: "Die Mumble Installation."
+url: "anleitungen/mumble"
+aliases:
+    - /de/anleitungen/mumble/
+    - /anleitungen/mumble/
+weight: 50
+---
+
+## Installation
+
+Informationen zu Mumble findest du über [mumble.sourceforge.net](http://mumble.sourceforge.net). Zur 
+[Installation](https://wiki.mumble.info/wiki/Installing_Mumble) musst du dir die passenden Dateien für dein 
+Betriebssystem herunterladen. Du benötigst nur den Mumble-Client. Die Installation der Mumble-Server Anwendung 
+ist nicht notwendig.
```

Comment:
The official website is https://www.mumble.info/ - may be link that, instead of sourceforge?

Reply:
Ok